### ISSUE TITLE
Change @ to another qualifier - Suggestion

### ISSUE THREAD

**Issue by leberschnitzel** [balzack/databag#32]
Right now the users are recognized as "username@server.tld"
If wide spread adoption is the goal, this will be confused with Email addresses very easily.

I suggest the use of another qualifier. Discord is getting rid of it's #, maybe that would be an idea?

**Comment by balzack** [comment]
I agree, and have struggled with this for awhile. Out of the following, is '#' your preference?

username~server.tld
username::server.tld
username#server.tld
username/server.tld

**Comment by leberschnitzel** [comment]
I like # because it's known and easy to type, but might also get other interpretation by people that use social media.
~ I think is the coolest, but at least in some languages it's annoying since if you type it, it doesn't just appear, you have to press another key after it. Don't know if that's the same in English?
/ makes sense and doesn't look bad, but I think it can make formatting problems when you want to share it somewhere.

Lot's of pros and cons. I think personally I like the / the most, since it's easy, makes some amount of sense, and the con is more for advanced users that will want to find a solution for the escaping problem anyway.

**Comment by balzack** [comment]
On a 'qwerty' keyboard the '~' is similar to the '@', it requires the shift key. I agree with your pros and cons, and will go with '/'. I'll include this in the release or two. Thank you for the feedback!

**Comment by lil5** [comment]
XMPP uses username@chat.example.com as well, I suggest keeping it how it is

**Comment by balzack** [comment]
The current mobile app allows you to login with either '@' or '/' as a separator, but within the app it displays the '/'.

Were you referring to this latest version, or should I avoid using '/' everywhere?

**Comment by mbaum0** [comment]
Any revelations on this? I think both `@` and `/` are fine, with a slight preference for `@`. As long as it's consistent

**Comment by balzack** [comment]
The primary issue people were facing was at login time, so to resolve this the login field will be separated into two fields { server, username }

Elsewhere in the app, it's used only for display purposes, so I don't have a preference. Since you like @, I will use that.
